src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala

```diff
@@ -0,0 +1,79 @@
+package cromwell.engine.backend.jes
+
+import java.security.MessageDigest
+
+import cromwell.binding.values._
+import cromwell.engine.EngineFunctions
+import cromwell.util.google.GoogleCloudStoragePath
+
+import org.apache.commons.codec.binary.Base64
+
+import scala.util.{Success, Try}
+
+/**
+ * Implementation of engine functions for the JES backend.
+ */
+case class JesEngineFunctions(callDir: GoogleCloudStoragePath, jesConnection: JesInterface) extends EngineFunctions {
```

Comment:
Just a heads-up regarding merging this to develop... I think I added a new function to EngineFunctions so you might have to just do `= ???` for that function definition to get it to compile